sdk/identity/identity/samples/AzureIdentityExamples.md

```diff
@@ -16,9 +16,29 @@ Authenticating your application, users, and principals is an integral part of wo
 
 ## Authenticating client side browser applications
 
-For client side applications running in the browser, the `InteractiveBrowserCredential` provides the simplest user authentication experience and is the only credential type that we support in the browser. To get started, you will want to configure an AAD application for interactive browser authentication. Please refer to the [Single-page application: App registration guide](https://docs.microsoft.com/azure/active-directory/develop/scenario-spa-app-registration) for additional information on how to configure your app registration for the browser.
+For client side applications running in the browser, the `InteractiveBrowserCredential` provides the simplest user authentication experience and is the only credential type that we support in the browser. To get started, you need to register your application in the Microsoft identity platform and set the right permissions.
 
-You may also refer to [Authenticating a user account interactively in the browser](#authenticating-a-user-account-interactively-in-the-browser) for an example of how you can use the `InteractiveBrowserCredential` once the app registration is configured.
+- [Register a single page application](https://docs.microsoft.com/azure/active-directory/develop/scenario-spa-app-registration) in the Microsoft identity platform
```

Comment:
```suggestion
- [Register a single page application](https://docs.microsoft.com/azure/active-directory/develop/scenario-spa-app-registration) in the Azure Active Directory.
```

Not sure if this is better? "identity platform" sounds a bit vague.

Reply:
I copied that from the official docs from docs.microsoft.com...

Reply:
Welp! That's ok then